geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseIndexAdapter.scala

```diff
@@ -451,20 +451,19 @@ class HBaseIndexAdapter(ds: HBaseDataStore) extends IndexAdapter[HBaseDataStore]
   private def groupRange(
       locator: RegionLocator,
       range: RowRange,
-      result: scala.collection.mutable.Map[ServerName, java.util.List[RowRange]]): Unit = {
-    var regionServer: ServerName = null
+      result: scala.collection.mutable.Map[String, java.util.List[RowRange]]): Unit = {
+    var regionInfo: HRegionInfo = null
```

Comment:
i'd change this to be the encoded name, so that if there's an exception getting the location it won't throw an NPE down on line 466/7